### PR TITLE
Current sketch will stop playing if a new example is opened (issue #357)

### PR DESCRIPTION
**Pull request by zrispo**

### CODE REVIEW

**Comment by catarak** [comment]
I'm not sure this is the right place to dispatch this `stopSketch` action. There's a larger issue here, which is that anytime a user opens a different sketch or navigates to a different page, the sketch should stop. Maybe this action should be dispatched with the route changes?

File: client/components/Nav.jsx [hunk 1]
```diff
@@ -91,7 +91,10 @@ class Nav extends React.PureComponent {
           })()}
           <li className="nav__item">
             <p className="nav__open">
-              <Link to="/p5/sketches">
+              <Link
+                to="/p5/sketches"
+                onClick={this.props.stopSketch}
```
Comment (catarak) [inline client/components/Nav.jsx]: with the `onRouteChange` handler I don't think you need this anymore!

Comment (zrispo) [inline client/components/Nav.jsx]: Oh that's right - there may be some extra `stopSketch` calls in the code, I'll go through and clean them up

**Comment by zrispo** [comment]
Got rid of the extra `stopSketch` calls, should be ready to go ~

**Pull request merged**